contracts/NFTs/IntegrationId.sol

```diff
@@ -0,0 +1,48 @@
+//SPDX-License-Identifier: BUSL-1.1
+pragma solidity ^0.8.13;
+
+import "./Base/NftBaseUpgradeable.sol";
+import "../interfaces/IDimoRegistry.sol";
+
+contract IntegrationId is Initializable, NftBaseUpgradeable {
```

Comment:
Is there any advantage to going up to multi-privilege now?

Reply:
Like if we want to add it later, are there going to be storage problems?

Reply:
Oh definetly. Might be worth adding in like we did with the aftermarket device even if we would not use it yet.